lib/board_transformer.rb

```diff
@@ -0,0 +1,42 @@
+module BoardTransformer
+    def horizontal_rows(board)
+        return board.positions.each_slice(board.size).to_a
+    end
+    
+    def vertical_rows(board)
```

Comment:
there's a simpler way to get columns... hint: look into `transpose`

Reply:
I think rows and columns are clearer names for horizontal_rows and vertical_rows - what do you think?

Reply:
Ok, changing!